tests/test_agent_orchestration_validation.py

```
@@ -0,0 +1,625 @@
+                timeout=context.timeout_seconds
+            )
+            assert False, "Should have timed out"
+        except asyncio.TimeoutError:
+            # Expected behavior
+            pass
+
+    @pytest.mark.asyncio
+    async def test_agent_error_isolation(self, mock_orchestrator):
+        """Test that agent errors don't affect other parallel executions."""
+        # Create mixed agent executions
+        async def mixed_agent_execution(agent_name: str, context: AgentExecutionContext):
```

Ruff (0.13.1)

Unused function argument: `context`

(ARG001)